module/Decision/src/Decision/Model/Meeting.php

```diff
@@ -214,4 +214,29 @@ public function addDocuments($documents)
         }
     }
 
+    /**
+     * Sorts document list
+     */
+    public function sortDocuments()
+    {
+        foreach ($this->documents as $document) {
+            $temp[] = $document;
+        }
+        usort($temp, function ($a, $b) {
+            $aa = preg_split("/(\.|\s)/", $a->getName());
+            $bb = preg_split("/(\.|\s)/", $b->getName());
+            for ($i = 0; $i < min(count($aa), count($bb)); $i++) {
+                if (!is_numeric($aa[$i])) {
+                    return -1;
+                } elseif (!is_numeric($bb[$i])) {
+                    return 1;
+                } elseif ($aa[$i] != $bb[$i]) {
+                    return $aa[$i] < $bb[$i] ? -1 : 1;
+                }
```

Comment:
Using `else` generally decreases readability in methods. Instead, I would rewrite it into:

```php
if (!is_numeric($aa[$i])) {
    return -1;
}
if (!is_numeric($bb[$i])) {
    return 1;
}
if ($aa[i] != $bb[i]) {
    return $aa[$i] < $bb[i]$ ? -1 : 1;
}
```

Also. This function seems like it's basically lexicographic sort on document names. Would sort on `$doc->getName()` not be good enough?

Reply:
That would break when there are more than 9 documents
1
10
11
12
2
3
4
5
6